src/index.js

```diff
@@ -6301,6 +6287,26 @@ export class ULabel {
         }
     }
 
+    // Update the displayed annotation confidence
+    update_confidence_dialog() {
+        // Whenever the mouse makes the dialogs show up, update the displayed annotation confidence.
+        const current_subtask = this.subtasks[this.state["current_subtask"]]
+        const active_annotation_id = current_subtask["active_annotation"]
+        const active_annotation = current_subtask["annotations"]["access"][active_annotation_id]
+        const aacp = active_annotation["classification_payloads"] // aacp stands for active annotation classification payloads
```

Comment:
```suggestion
        /** The active annotation's classification payloads. */
        const aacp = active_annotation["classification_payloads"]
```
Two changes here: imperative phrasing, and JSDoc syntax (see [this](https://stackoverflow.com/questions/38708777/what-is-the-correct-jsdoc-syntax-for-a-local-variable)).
JSDoc will allow editors to render the description on hover like this:
![image](https://github.com/SenteraLLC/ulabel/assets/7959279/5eeb525b-ee07-4edc-89e6-cfb5c3b16f8d)

This is useful for locals like this that are named as an abbreviation of something much more verbose. 
I wouldn't advocate mandatory JSDoc on all locals.

Reply:
Yeah that's definitely a good call :+1: